coq/coq-syntax.el

```diff
@@ -1297,6 +1297,9 @@ It is used:
   "*Font-lock table for Coq terms.")
 
 
+(defconst coq-show-proof-diffs-regexp
+  "^[ ]*Show Proof\\( Diffs\\| Diffs removed\\)?\\.$")
```

Comment:
```suggestion
  "\\`Show Proof\\(?: Diffs\\| Diffs removed\\)?\\.\\'")
```
Indeed
* `[ ]*` seems unnecessary as we just want to detect the "Show Proof…" commands that PG itself generates (so without leading spaces)
* the `\\(?:` marks the group "non-capturing"
* and finally \\` and \\' looks better to anchor the string

(according to https://www.gnu.org/software/emacs/manual/html_node/elisp/Syntax-of-Regexps.html#Syntax-of-Regexps )

**Note:** please re-test that regexp manually as well, with `(string-match-p coq-show-proof-diffs-regexp "…string…")`

Reply:
\` doesn't seem to work, \b seems more appropriate